director/projects/source_views.py

```diff
@@ -426,10 +426,10 @@ def convert_source_for_google_docs(self, scf: SourceContentFacade) -> typing.Tup
             with open(temp_output_path, 'r+b') as temp_output:  # reopen after data has been written
                 output_content = temp_output.read()  # this is in DOCX after conversion
         finally:
-            if use_temp_input_path and input_path:
```

Comment:
It needs to still check that input_path is set, so:
```if use_temp_input_path and input_path and os.path.exists(input_path):```
If `input_path` is `None` a `TypeError` is raised.

Reply:
Same as below.